.github/container/get-source.sh

```diff
@@ -1,55 +1,54 @@
 #!/bin/bash
-## Fetch a Python package from a git repo and write the pip-tools input manifest to stdout
+## Clone a git repo and write the pip-compile input to stdout
 ## Example:
-## get-source.sh -f https://github.com/google/flax.git -r main -d /opt/flax
+## get-source.sh -m manifest.yaml -l flax
 ## Output:
 ## -e /opt/flax
 
 ## Parse command-line arguments
 
 usage() {
     echo "Usage: $0 [OPTION]..."
```

Comment:
[minor] I think it is better to refactor it in the following manner:
```
usage() {
cat<<OEF
   <Description of the script: what it does, what is the purpose, another joke>
   Usage: $0 [OPTION]
...
```

Reply:
Done